ui/app/src/components/ItemActionsMenu/ItemActionsMenu.tsx

```diff
@@ -78,7 +78,8 @@ export function ItemActionsMenu(props: ItemMenuProps) {
     clipboard &&
     clipboard.paths.length &&
     getRootPath(clipboard.sourcePath) === getRootPath(item.path) &&
-    isValidCutPastePath(item.path, clipboard.sourcePath);
+    isValidCopyPastePath(item.path, clipboard.sourcePath) &&
+    (clipboard.type === 'CUT' ? isValidCutPastePath(item.path, clipboard.sourcePath) : true);
```

Comment:
If it's not copy paste, why check for valid copy paste?
```suggestion
    (clipboard.type === 'CUT' ? isValidCutPastePath(item.path, clipboard.sourcePath) : isValidCopyPastePath(item.path, clipboard.sourcePath));
```